src/main/java/seedu/address/logic/commands/contact/EditContactCommand.java

```diff
@@ -126,35 +128,35 @@ public boolean equals(Object other) {
 
         EditContactCommand otherEditCommand = (EditContactCommand) other;
         return index.equals(otherEditCommand.index)
-                && editPersonDescriptor.equals(otherEditCommand.editPersonDescriptor);
+                && editContactDescriptor.equals(otherEditCommand.editContactDescriptor);
     }
 
     @Override
     public String toString() {
         return new ToStringBuilder(this)
                 .add("index", index)
-                .add("editPersonDescriptor", editPersonDescriptor)
+                .add("editContactDescriptor", editContactDescriptor)
                 .toString();
     }
 
     /**
      * Stores the details to edit the contact with. Each non-empty field value will replace the
      * corresponding field value of the contact.
```

Comment:
should there be a line break after `the`?

Reply:
did not change the comments